__tests__/unit/validators/description.test.js

```diff
@@ -30,6 +30,12 @@ test('isMergeable is false if the PR body is empty', async () => {
 
   descriptionValidation = await description.processValidate(createMockPR('Some Description'), settings)
   expect(descriptionValidation.status).toBe('pass')
+  
+  descriptionValidation = await description.processValidate(createMockPR(undefined), settings)
+  expect(descriptionValidation.status).toBe('pass')
```

Comment:
shouldn't this be 'fail'? since `no_empty` is `enabled: true`

Reply:
@shine2lay yup, my bad. I changed it. Please check. Thanks!